libbeat/processors/add_docker_metadata/add_docker_metadata_test.go

```diff
@@ -134,13 +134,9 @@ func TestMatchContainer(t *testing.T) {
 				"id":    "container_id",
 				"image": "image",
 				"labels": common.MapStr{
-					"a": common.MapStr{
-						"x": "1",
-					},
-					"b": common.MapStr{
-						"value": "2",
-						"foo":   "3",
-					},
+					"a_x":   "1",
```

Comment:
If test outputs changed, I assume it's a breaking change and should also be noted like this in the changelog?

Reply:
With suggestions from @jsoriano , I'm adding dedot into config and default it to be false. So this test output should not be changed and also no breaking change here.